modules/microshift-oc-mirror-list-ops-catalogs.adoc

```diff
@@ -0,0 +1,47 @@
+//Module included in the following assemblies:
+//
+//* microshift_running_apps/microshift-operators-olm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-oc-mirror-list-operators-catalogs_{context}"]
+= Using oc mirror to select Operators
+
+Use the following example procedure to select a catalog and Operators to add to your oc mirror image set configuration file.
+
+.Prerequisites
+* The {oc-first} is installed.
+* Operator Lifecycle Manager (OLM) is installed.
+* The oc-mirror OpenShift CLI (oc) plugin is installed.
+
+.Procedure
+. Get a list of available catalogs to filter by running the following command:
++
+[source,terminal]
+----
+$ oc mirror list operators --version 4.15 --catalogs
+----
+//Q: can we use the --version 4.15 --catalogs flags for homemade Operators? Do we need to have loaded them onto oc mirror in a previous step?
+
+. Get a list of Operators in the catalog by running the following command:
++
+[source,terminal]
+----
+$ oc mirror list operators --catalog=registry.redhat.io/redhat/redhat-operator-index:v4.15
+----
+//Q: is --catalog=registry.redhat.io/redhat/redhat-operator-index:v4.15 a replaceable value for homemade and homeserved catalogs?
```

Comment:
Yes, I think it should work for the file-based catalogs.
Example using upstream OperatorHub.io's catalog:
```
$ oc mirror list operators --catalog=quay.io/operatorhubio/catalog:latest

Logging to .oc-mirror.log
NAME                                      DISPLAY NAME                                                              DEFAULT CHANNEL
ack-acm-controller                        AWS Controllers for Kubernetes - Amazon ACM                               alpha
ack-acmpca-controller                     AWS Controllers for Kubernetes - Amazon ACM PCA                           alpha
ack-apigatewayv2-controller               AWS Controllers for Kubernetes - Amazon API Gateway v2                    alpha
ack-applicationautoscaling-controller     AWS Controllers for Kubernetes - Amazon Application Auto Scaling          alpha
ack-cloudfront-controller                 AWS Controllers for Kubernetes - Amazon CloudFront                        alpha
ack-cloudtrail-controller                 AWS Controllers for Kubernetes - Amazon CloudTrail                        alpha
ack-cloudwatch-controller                 AWS Controllers for Kubernetes - Amazon CloudWatch                        alpha
ack-cloudwatchlogs-controller             AWS Controllers for Kubernetes - Amazon CloudWatch Logs                   alpha
ack-dynamodb-controller                   AWS Controllers for Kubernetes - Amazon DynamoDB                          alpha
ack-ec2-controller                        AWS Controllers for Kubernetes - Amazon EC2                               alpha
...
```